src/mobile/pages/fields/field/field.js

```diff
@@ -328,6 +367,10 @@ export const FieldVM = DefineMap.extend('FieldVM', {
       this.debugPanelMessage(field, value)
     }
 
+    if (((field.type === 'datemdy') && (value.length < 6))) {
```

Comment:
should check if length is either 6 or 8 digits not only less than 6. This will pass 7 and 9+ which is invalid.